Adding ML-Genetic Algorithm example (#2981)
ansys/pymapdl: doc/source/user_guide/hpc_slurm.rst

    Number of CPUs: 10.0

Examples
========

> In :ref:`hpc_ml_ga_example` an example which uses machine learning genetic algorithm in
> an HPC system managed by SLURM scheduler can be found.

Reviewer: ```suggestion
For an example that uses a machine learning genetic algorithm in
an HPC system managed by SLURM scheduler, see :ref:`hpc_ml_ga_example`.
```
Perhaps you can use GA here for genetic algorithm. (I don't know if it's been defined before its use here. Once defined, you can use GA thereafter.

> Monitoring Jobs
> ===============

Reviewer: ```suggestion
Monitor jobs
============

The following commands provide for monitoring jobs.
```
